cpp/CMakePresets.json

```diff
@@ -6,6 +6,30 @@
     "patch": 0
   },
   "configurePresets": [
+    {
+      "name": "_allocator-none",
```

Comment:
It seems that we can add comments by `"$comment"`: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#format

Could you add a comment why we use `_XXX` name here?

Reply:
Thank you for the idea. But unfortunately `"$comment"` is only supported in version 10 or above, and version 10 is only supported since cmake 3.31.

From the link you provided.

> Preset files specifying version 10 or above may include comments using the key $comment at any level within the JSON object to provide documentation.

> 10  Added in version 3.31.

Reply:
Oh...